From dd914da1c80941ed1c435671cac20ca4f5f7f0e6 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Thu, 16 Mar 2017 09:12:26 -0400
Subject: [PATCH] gio/fam: Remove leftover debug print

This leftover debug print was introduced by d682df186e9cca2b8db0e921450a4db31cf0c467
and is obviously bad for applications that are expecting something
else on stdout, etc.

See: https://bugzilla.redhat.com/show_bug.cgi?id=1396386

https://bugzilla.gnome.org/show_bug.cgi?id=780144
---
 gio/fam/gfamfilemonitor.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/gio/fam/gfamfilemonitor.c b/gio/fam/gfamfilemonitor.c
index 50474c8..551b3c2 100644
--- a/gio/fam/gfamfilemonitor.c
+++ b/gio/fam/gfamfilemonitor.c
@@ -140,8 +140,6 @@ g_fam_file_monitor_is_supported (void)
 
   g_mutex_unlock (&fam_lock);
 
-  g_print ("II %d\n", fam_initialised);
-
   return fam_initialised;
 }
 
-- 
2.9.3

